test/express-route.test.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var path = require('path'),
+  helpers = require('yeoman-generator').test,
+  assert = require('yeoman-generator').assert,
+  temp = require('temp').track();
+
+describe('Express Sub Generators Tests', function () {
+  this.timeout(0);
```

Comment:
CI tests failed with timeouts, and the `app.test.js` already had the timeout disabled (probably, because git), so I went in the same direction.

Strictly speaking, timeout testing is not really applicable for the generators anyways - the speed is much more dependent on the file system, than on the actual code itself. So, the timeouts were disabled.